models/Event.ts

```diff
@@ -41,11 +41,11 @@ export interface QueryOptions {
 export interface EventStatic extends ModelStaticCommon<Event> {
   query: (
     user: User,
-    startTime: string,
-    endTime: string,
-    deviceId: DeviceId,
-    offset: number,
-    limit: number,
+    startTime: string | null,
```

Comment:
I would have expected the actual type if the variable is not defined to be `undefined`, rather than `null`